notebooker/web/app.py

```diff
@@ -138,6 +138,8 @@ def main(web_config: WebappConfig):
     GLOBAL_CONFIG = web_config
     flask_app = create_app(web_config)
     flask_app = setup_app(flask_app, web_config)
+    serializer = get_serializer_from_cls(web_config.SERIALIZER_CLS, **web_config.SERIALIZER_CONFIG)
+    serializer.enable_sharding()
```

Comment:
I think this is fine, but we need to make a note in the CHANGELOG that this will break any custom serializer which doesn't implement the `enable_sharding()` method. We don't provide a base class apart from the MongoResultSerializer so this is probably okay, but worth mentioning that this could cause issues.